Entity-ngsiv2-schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-schema.json",
+  "title": "NGSIv2 Entity",
+  "description": "NGSIv2 Entity",
+  "definitions": {
+    "EntityIdentifier": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$"
+    },
+    "Name": {
```

Comment:
This "Name" is for attribute names and metadata names?

Take into account the following sections in [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable):

"Field syntax restrictions"
"Attribute names restrictions"
"Metadata names restrictions"

For instance, "id" and "type" literals are forbidden, there is a max lenght, * is forbidden, etc.

Reply:
In addition, pattern should be the same that for entities id (currently it isn't)

Reply:
Maybe what can be done (assuming JSON Schema allows some form of inheritance, which I don't know) is to define a a base Identifier type with the restrictions in "Field syntax restrictions" then derive from it an AttributeName (adding the "Attribute names restrictions") and a MetadataName type (adding "Metadata names restrictions").

Reply:
fixed https://github.com/Fiware/dataModels/pull/373/commits/4599e4602ea40bb543bff6338fed72ee99806b60